docs/List.md

```diff
@@ -1266,3 +1266,23 @@ const ProductList = () => {
 ```
 
 `useListController` returns callbacks to sort, filter, and paginate the list, so you can build a complete List page. Check [the `useListController`hook documentation](./useListController.md) for details.
+
+## Controlling Access To The List
+
+### Controlling Access In Exports
+
+Should your authProvider implements the [`canAccess` method](./AuthProviderWriting.md#canaccess), the [`exporter`](#exporter) will call it for each record property to ensure it only exports fields to which users have access to. For instance, given the following record shape:
+
+```json
+{
+    "id": 0,
+    "title": "Never Gonna Give You Up",
+    "author": "Rick Astley",
+}
+```
+
+The `authProvider.canAccess` method will be called 3 times with the following parameters:
+
+1. `{ action: "read", resource: "posts.id" }`
```

Comment:
I think we should use the `export` action for that. This allows to have a distinction between fields that are shown and the ones that are exported.